the-forest.kvp

```diff
@@ -1,35 +1,87 @@
+App.EnvironmentVariables={"LD_LIBRARY_PATH":"./linux64:%LD_LIBRARY_PATH%","SteamAppId":"242760"}
+App.CommandLineParameterFormat=-{0} "{1}"
+App.CommandLineParameterDelimiter= 
+App.ExitMethod=OS_CLOSE
+App.ExitTimeout=30
+App.ExitString=stop
+App.ExitFile=app_exit.lck
+App.HasWriteableConsole=True
+App.HasReadableConsole=True
+App.SupportsLiveSettingsChanges=False
+App.LiveSettingChangeCommandFormat=set {0} "{1}"
+App.ApplicationIPBinding=0.0.0.0
+App.Ports=[{"Protocol":2,"Port":27015,"Offset":0,"Range":1,"Ref":"gamePort","Name":"gamePort","Description":"Main Game Port","ChildPorts":null},{"Protocol":2,"Port":8766,"Offset":0,"Range":1,"Ref":"steamPort","Name":"steamPort","Description":"Steam Main Port","ChildPorts":null},{"Protocol":2,"Port":27016,"Offset":0,"Range":1,"Ref":"queryPort","Name":"queryPort","Description":"Steam Query Port","ChildPorts":null}]
+App.AdminPortRef=RemoteAdminPort
+App.PrimaryApplicationPortRef=gamePort
+App.UniversalSleepApplicationUDPPortRef=gamePort
+App.UniversalSleepSteamQueryPortRef=queryPort
+App.MaxUsers=4
+App.UseRandomAdminPassword=False
+App.RemoteAdminPassword=Password123
+App.AdminMethod=STDIO
+App.AdminLoginTransform=None
+App.RCONConnectDelaySeconds=5
+App.RCONConnectRetrySeconds=5
+App.RCONHeartbeatMinutes=0
+App.RCONHeartbeatCommand=ping
+App.TelnetLoginFormat={0}
+App.UpdateSources=[{"UpdateStageName": "SteamCMD Download","UpdateSourcePlatform": "All", "UpdateSource": "SteamCMD", "UpdateSourceData": "556450", "ForceDownloadPlatform": "Windows"},{"UpdateStageName": "Base Config Download","UpdateSourcePlatform": "All","UpdateSource": "FetchURL","UpdateSourceData": "https://github.com/IceOfWraith/AMPTemplates/raw/the-forest/the-forestserver.cfg","UpdateSourceTarget": "./556450/"}]
+App.Compatibility=None
+App.SteamUpdateAnonymousLogin=True
+App.SteamForceLoginPrompt=False
+App.RapidStartup=False
+App.MonitorChildProcess=True
+App.MonitorChildProcessWaitMs=1000
+App.MonitorChildProcessName=wine64
+App.SupportsUniversalSleep=False
+App.WakeupMode=Any
+App.ApplicationReadyMode=RegexMatch
+App.TemplateMatchRegex={{(\$?[\w]+)}}
+Console.FilterMatchRegex=
+Console.FilterMatchReplacement=
 Console.ThrowawayMessageRegex=^(WARNING|ERROR): Shader.+$
-Console.UserChatRegex=^(?<username>.+?): (?<message>.+)$
-Console.UserJoinRegex=^User (?<username>.+?) \((?<userid>-?d+)\) connected from \[::ffff:(?<endpoint>.+?)\]$
-Console.UserLeaveRegex=^User (?<username>.+?) \((?<userid>-?d+)\) disconnected\. Reason: (.+?)$
-Meta.Author=CubeCoders Limited
-Meta.ConfigManifest=the-forestconfig.json
-Meta.ConfigRoot=the-forest.kvp
-Meta.Description=
-Meta.DisplayImageSource=steam:242760
-Meta.DisplayName=The Forest
-Meta.EndpointURIFormat=steam://connect/{ip}/{GenericModule.App.ApplicationPort3}
-Meta.OS=1
-Meta.URL=https://store.steampowered.com/app/242760/The_Forest/
\ No newline at end of file
+Console.AppReadyRegex=^GameServer init success.*$
+Console.UserJoinRegex= 
+Console.UserLeaveRegex= 
+Console.UserChatRegex= 
```

Comment:
Suggest this be "zeroed" with ^$ to avoid unexpected matching of AMP's default regex